lttngpy/src/lttngpy/_lttngpy_pybind11.cpp

```diff
@@ -90,6 +107,12 @@ PYBIND11_MODULE(_lttngpy_pybind11, m) {
     "Stop tracing.",
     py::kw_only(),
     py::arg("session_name"));
+  m.def(
+    "lttng_record_snapshot",
+    &lttngpy::lttng_record_snapshot,
```

Comment:
Similar to my other comment

```suggestion
    "record_snapshot",
    &lttngpy::record_snapshot,
```